### DISCUSSION TITLE
Ability to clone transactions in v2 UI?

### DISCUSSION THREAD

**Discussion by Dreytac** [firefly-iii/firefly-iii#5122] [Firefly III] [answered]
I've noticed in the latest update to the account page that the "clone" option under the actions menu on a transaction is gone. If I click edit, there's no clone option on that page either. I made frequent use of the clone function in that menu.

The only way I see to clone a transaction now is to click on the description of a transaction (which takes me to the v1 UI), click the actions drop down and select clone from there.

Is there any intention of adding the "clone" option anywhere under the new v2 UI (or an I missing something completely)?

**Comment by JC5** [comment, marked as answer]
Yes there is, and since it's a beta layout such features may be implemented at a later date.
